packages/@react-aria/checkbox/test/useCheckboxGroup.test.tsx

```diff
@@ -160,7 +159,7 @@ describe('useCheckboxGroup', () => {
   });
 
   it('sets aria-disabled when isDisabled is true', () => {
```

Comment:
I believe that those test names should be adjusted now as a little more than just `aria-disabled` is being tested here